README.md

```diff
@@ -59,7 +59,7 @@ Activate a version you've already installed.
 nvm use v14
 ```
 
-List which versions you have installed (includes your system-installed Node if there is one).
+List which versions you have installed (includes your system-installed Node if there is one), which are stored under `$nvm_data`.
```

Comment:
I'd like to remove this extra info. Nice effort, but it makes the sentence harder to read. @kidonng

Reply:
It's fine too, given that:
- `$nvm_data` is set to `$XDG_DATA_HOME/nvm` unconditionally when the plugin installs/updates, and doesn't to customize it
- `$XDG_DATA_HOME/nvm` is pretty is a natural and predictable location